examples/custom_response_class.py

```diff
@@ -0,0 +1,14 @@
+from curl_cffi import requests
+
+class CustomResponse(requests.Response):
+    @property
```

Comment:
Coud you please add an `__init__` method that extracts other values from the `.curl` property.

Something like this:

https://github.com/lexiforest/curl_cffi/blob/137a83451e8032d6ffaed1f54fa605f828760309/curl_cffi/requests/session.py#L706-L707

For a list of getinfo options, see here: https://curl.se/libcurl/c/easy_getinfo_options.html

Reply:
Done. By the way, any `CurlInfo` of `0x600000` (such as `SIZE_UPLOAD_T`) aren't supported in this dict:
https://github.com/lexiforest/curl_cffi/blob/137a83451e8032d6ffaed1f54fa605f828760309/curl_cffi/curl.py#L232-L237
And cause a `KeyError` when reaching:
https://github.com/lexiforest/curl_cffi/blob/137a83451e8032d6ffaed1f54fa605f828760309/curl_cffi/curl.py#L243